exercises/nth-prime/.meta/additional_tests.json

```diff
@@ -0,0 +1,13 @@
+{
+  "cases": [
+    {
+      "description": "first twenty primes",
+      "property": "prime range",
```

Comment:
```suggestion
      "property": "primeRange",
```

Reply:
A bit nit-picky, but the canonical data uses camelCase for its test names. The idea is to keep additional_tests.json as close to the format of canonical-data.json as possible.